xla/pjrt/cpu/gloo_collectives_test.cc

```diff
@@ -25,8 +25,12 @@ limitations under the License.
 #include "absl/status/statusor.h"
 #include "absl/time/time.h"
 #include "absl/types/span.h"
+#ifdef __linux__
 #include "gloo/transport/tcp/attr.h"
 #include "gloo/transport/tcp/device.h"
+#elif __APPLE__
```

Comment:
Should this be `#elifdef`? Other places in XLA also check if it's defined.
```suggestion
#elifdef __APPLE__
```